system-tests/projects/config-cjs-and-esm/config-with-ts-module/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Any idea why the linter did not get run against this? Either way that's an easy one to clear up 😅

Reply:
Something in here I think... https://github.com/cypress-io/cypress/blob/develop/.eslintignore
Cleaned it up :+1: